packages/office-addin-debugging/src/start.ts

```diff
@@ -172,28 +172,95 @@ export async function runPackager(commandLine: string, host: string = "localhost
     }
 }
 
+
 /**
  * Start debugging
- * @param manifestPath The path to the manifest file.
- * @param appType The type of application to debug.
- * @param app The Office application to debug.
- * @param debuggingMethod The method to use when debugging.
- * @param sourceBundleUrlComponents Specify components of the source bundle url.
- * @param devServerCommandLine If provided, starts the dev server.
- * @param devServerPort If provided, port to verify that the dev server is running.
- * @param documentUrl If provided, the document to open for sideloading to web.
- * @param packagerCommandLine If provided, starts the packager.
- * @param packagerHost Specifies the host name of the packager.
- * @param packagerPort Specifies the port of the packager.
- * @param enableDebugging If false, start without debugging.
+ * @param options start debugging options.
  */
-export async function startDebugging(manifestPath: string, appType: AppType, app: OfficeApp | undefined,
```

Comment:
@akrantz How come the `app` variable is allowed to be undefined? Should `app` be a required parameter?

Reply:
No, we will prompt the user if not specified.

Reply:
When you use the CLI, you can do "office-addin-debugging start <manifest>" and it if there is more than one app in the manifest, it will prompt for which one, or the user can add `--app excel` to specify. If there's only one app in the manifest, it doesn't bother prompting. The VS Code configs specify the app, since you need a different launch target for each app.

Reply:
I don't think you should try to change whether or not undefined is allowed for particular parameters here or the defaults. If you see some that you really think we should change, we can discuss it.

Reply:
I see, perhaps the documentation should be updated to specify that if `app` is undefined it will prompt to specify the application?